docs/updates/2019-04.md

```diff
@@ -0,0 +1,180 @@
+# April 2019
+
+Welcome to the April 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Enhancements in catwalk
+
+### Demo app
+
+Looking at the visitor statistics we could see that a lot of people opened and left catwalk without opening a Qlik
+application. One possible explanation to the behavior could be a curiosity around the tool and its capabilities but
```

Comment:
Rephrase, sounds like we're guessing :) Maybe "One possible explanation to this behavior is that these visitors are curious about the tool, but [...]"

Reply:
Aren't we ;-)

Reply:
We're not double guessing at least :) "Possible explanation", "could be"